documentation/source/users/rmg/database/modification.rst

```diff
@@ -73,6 +72,167 @@ There are several places in the RMG-database and RMG-Py source code where reacti
 Creating Kinetics Libraries
 ---------------------------
 
+To add a reaction library, simply create a folder bearing the library's name under
+``RMG-database/input/kinetics/libraries``. You'll need to create two files:
+``dictionary.txt`` and ``reactions.py``. The dictionary file contains the Adjacency lists
+for all relevant species (can be generated using the `Molecule Search <http://rmg.mit.edu/molecule_search>`_
+function of the rmg website, while the reactions file specifies the kinetics.
+To conform to RMG's format, simply copy and modify an existing library.
+
+At the top of the reactions file fill in the name and short (one line) and long descriptions.
+The name must be identical to the folder's name. Then list the kinetics entries, each with a unique index number.
+The following formats are accepted as kinetics entries:
+
+**Arrhenius** of the form :math:`k(T) = A \left( \frac{T}{T_0} \right)^n \exp \left( -\frac{E_\mathrm{a}}{RT} \right)`
+(see `Arrhenius Class <http://reactionmechanismgenerator.github.io/RMG-Py/reference/kinetics/arrhenius.html>`_ for details)::
+
+	entry(
+        index = 1,
+    	label = "H + O2 <=> O + OH",
+    	degeneracy = 1,
+    	kinetics = Arrhenius(A=(9.841e+13, 'cm^3/(mol*s)'), n=0, Ea=(15310, 'cal/mol'), T0=(1, 'K')),
+    	shortDesc = u"This is a short description limited to one line, e.g. 'CBS-QB3'",
+    	longDesc = u"""This is a long description, unlimited by number of lines.
+    	These descriptions can be added to every kinetics type.""")
+
+**MultiArrhenius**
+(see `MultiArrhenius Class <http://reactionmechanismgenerator.github.io/RMG-Py/reference/kinetics/multiarrhenius.html>`_ for details)::
+
```

Comment:
Can you specify that MultiArrhenius is the sum of multiple Arrhenius expressions (as opposed to being one Arrhenius over one Trange and one Arrhenius over another Trange)